app/src/main/java/com/portfolio/tasky/networking/domain/RequestInterceptor.kt

```diff
@@ -0,0 +1,9 @@
+package com.portfolio.tasky.networking.domain
+
+import okhttp3.Interceptor
+import okhttp3.Response
+
+interface RequestInterceptor : Interceptor {
+
+    fun interceptRequest(chain : Interceptor.Chain) : Response
+}
```

Comment:
The intercepting will only happen on Retrofit

So basically, on the data layer. Domain shouldn't know about it
So no need to have an interface for this. You can completely remove this file

Reply:
Done